adapters/handlers/graphql/local/get/class_builder.go

```diff
@@ -177,6 +177,63 @@ func (b *classBuilder) additionalFields(classProperties graphql.Fields, class *m
 			Fields: additionalProperties,
 		}),
 	}
+
+	classProperties["groupedBy"] = &graphql.Field{
```

Comment:
I think that this needs to be removed... here we add `groupedBy` field also at the same level as `_additional` parameter and I think that this should not be here

Reply:
`groupedBy` field is part of `_additional.group.groupeBy` field it's not meant to be outside of `group`

Reply:
Removed